Automattic/jetpack
# mirrors: Enable auto-publishing to npmjs.com

#### Changes proposed in this Pull Request:

As the RNA stuff gets closer to completion, we're going to want to
publish those packages to npmjs.com. This creates the necessary
infrastructure.

This also sets up our eslint-changed tool for mirroring, auto-tagging,
and auto-publishing so it can be used as a test case.

Note a mirror repo for eslint-changed will need to be created before this is merged.

#### Jetpack product discussion


p9dueE-3eM-p2

#### Does this pull request change what data or activity we track or use?


no

#### Testing instructions:






* Check that the Build artifact includes eslint-changed. Check that it looks right, and in particular doesn't have an `.engines` in `package.json`.
* Download and unpack the artifact. Then, in the eslint-changed directory, run `npm publish --dry-run`. Verify that the list of files to be published by that command is correct, and in particular does not contain `.github/*` or `composer.json`.
